Writerside/topics/Quickstart-Up-And-Running.md

```diff
@@ -0,0 +1,585 @@
+
+<procedure title="Lyra Game Server Installation" id="lyra_setup" collapsible="true">
+    <p>These instructions use the <a href="https://github.com/catenatools/catena-lyra-demo">catena-lyra-demo</a> server.</p>
+    <step>Download the server onto the machine. The download can be found <a href="https://catena-public-content.S3.amazonaws.com/WindowsServer.zip">here.</a></step>
+    <step>Extract the zip file.</step>
+    <step>Run the game server from command prompt: <pre>LyraServer.exe -networkversionoverride=1234 -BackendUrl=localhost:8085</pre></step>
+    <step>Now the game server will communicate with the backend through the sidecar.</step>
+</procedure>
+
+## Run the Server Sidecar
+
+The server sidecar enables Catena to be aware of your game server without requiring any changes to the code of the game server itself. It is additionally able to make requests on behalf of your game server
+if you so choose. The sidecar will augment connection details and can translate requests between different backends so your server only needs to implement one SDK interface. See [Sidecar](Sidecar.md) for more details.
+
+First download the Sidecar to the machine, the latest release can be found [here](https://github.com/CatenaTools/sidecar/releases/). Alternatively, clone and build the sidecar.
+
+In the same directory as the sidecar, add a `config.json.` There is an example in the  sidecar repo here: https://github.com/CatenaTools/sidecar/blob/main/config.json
+
+```json
+{
+  "motd": "Hello Catena",
+  "app": {
+    "logging": {
+      "level": "debug",
+      "encoder": "console"
+    },
+    "event_source": {
+      "use": "catena_api_event_source",
+      "configs": {
+        "catena_api_event_source": {
+          "GrpcListenAddr": "0.0.0.0:8080",
+          "HttpListenAddr": ":8085"
+        },
+        "virtual_server_event_source": {
+          "strategy": "continuous_strategy",
+          "continuous_strategy": {
+            "RequestInterval": "5s"
+          }
+        }
+      }
+    },
+    "resolver": {
+      "use": "ec2_resolver",
+      "configs": {
+        "ec2_resolver": {
+          "MetadataBaseUrl": "http://169.254.169.254",
+          "AvailablePorts": [
+            {
+              "Min": 7777,
+              "Max": 7777
+            }
+          ]
+        },
+        "static_resolver": {
+          "Address": "server.local:8080"
+        }
+      }
+    },
+    "backend": {
+      "configs": {
+        "catena_backend": {
+          "url": "$CATENA_URL"
+        },
+        "noop_backend": {}
+      },
+      "use": "catena_backend"
+    },
+    "sidecar": {
+      "DebugPanelAddr": "localhost:8081",
+      "MatchManagementStrategy": {
+        "OverrideConnectionDetails": true,
+        "AutoPopulateIDs": true
+      }
+    }
+  }
+}
+```
+
+The key is to set the following fields:
+
+`app.backend.catena_backend.url` field, which must point to your deployed instance url.
```

Comment:
> field, which

leftover from a previous version? Reads a little odd and is different from the subsequent lines.